src/views/ipv/page/pyi-triage-select-smartphone.njk

```diff
@@ -0,0 +1,83 @@
+{% extends "shared/base.njk" %}
+{% from "govuk/components/button/macro.njk" import govukButton %}
+{% from "govuk/components/radios/macro.njk" import govukRadios %}
+{% from "govuk/components/details/macro.njk" import govukDetails %}
+
+{% set pageTitleKey = 'pages.pyiTriageSelectSmartphone.title' %}
+
+{% set googleTagManagerPageId = "pyiTriageSelectSmartphone" %}
+
+{% set errorState = pageErrorState %}
+{% set errorTitle = 'pages.pyiTriageSelectSmartphone.content.formErrorMessage.errorSummaryTitleText' | translate %}
+{% set errorText = 'pages.pyiTriageSelectSmartphone.content.formErrorMessage.errorSummaryDescriptionText' | translate %}
+{% set errorHref = "#pyiTriageSelectSmartphoneOptionsForm" %}
+
+{% block content %}
+  <h1 class="govuk-heading-l" id="header" data-page="{{ googleTagManagerPageId }}">{{ 'pages.pyiTriageSelectSmartphone.header' | translate }}</h1>
+  <p class="govuk-body">{{ 'pages.pyiTriageSelectSmartphone.content.paragraphIntro1' | translate }}</p>
+  <p class="govuk-body">{{ 'pages.pyiTriageSelectSmartphone.content.paragraphIntro2' | translate }}</p>
+
+  <h2 class="govuk-heading-m govuk-!-margin-top-7">{{ 'pages.pyiTriageSelectSmartphone.content.iphone.subHeading' | translate }}</h2>
+  <p class="govuk-body govuk-!-margin-bottom-2">{{ 'pages.pyiTriageSelectSmartphone.content.iphone.paragraph' | translate }}</p>
```

Comment:
Is this necessary `govuk-!-margin-bottom-2`? I think its causing a big gap in-between the text when displayed on the page. Looks like another occurrence on the page to

Reply:
Which gap do you mean? I think `govuk-!-margin-top-7` might be to blame for the biggest gap above the iPhone subheading. I've removed them now:
<img width="468" alt="Screenshot 2024-03-19 at 11 37 38" src="https://github.com/govuk-one-login/ipv-core-front/assets/144116535/d846d5d8-f2e2-40e5-95a3-7de942ce1ef4">